### PR TITLE
Completed phase 1 of react-design exercise

### PR DESCRIPTION
**Pull request by marshall-vogt**
Screenshot 2023-08-31 at 9 54 44 AM

### CODE REVIEW

File: react-design/my-app/src/RotatingBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export default function RotatingBanner({ items }) {
+  return (
+    <div className="container">
+      <div className="row">
+        <Banner item={items[0]} />
+      </div>
+      <div className="row">
+        <PreviousButton />
+      </div>
+      <div className="row">
```
Comment (RobertGardner) [inline react-design/my-app/src/RotatingBanner.tsx:30]: I suggest you put all these NumberButtons into a single component.

**Review by RobertGardner — COMMENTED** [review]
Looking good. Look at my suggestion, though.
Go ahead and proceed to the next phase.

File: react-design/my-app/src/NumberButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export default function NumberButtons(/* { count, index } */) {
```
Comment (RobertGardner) [inline react-design/my-app/src/NumberButtons.tsx]: In this phase, you should make these props "live".

**Review by RobertGardner — COMMENTED** [review]
Just one comment.

**Comment by marshall-vogt** [comment]
Kapture 2023-08-31 at 14 08 12

File: react-design/my-app/src/NumberButtons.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export default function NumberButtons({ count, current, onIndicatorClick }) {
+  const buttonArray = [];
+  for (let i = 0; i < count; i++) {
+    buttonArray.push(
+      <button
+        type="button"
+        className="column-sixth button"
+        key={i}
```
Comment (RobertGardner) [inline react-design/my-app/src/NumberButtons.jsx:8]: Good catch!

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**